easybuild/tools/module_naming_scheme/toolchain.py

```diff
@@ -89,8 +88,8 @@ def det_toolchain_compilers(ec):
     :param ec: a parsed EasyConfig file (an AttributeError will occur if a simple dict is passed)
     """
     tc_elems = ec.toolchain.definition()
-    if ec.toolchain.name == DUMMY_TOOLCHAIN_NAME:
-        # dummy toolchain has no compiler
+    if ec.toolchain.is_system_toolchain():
```

Comment:
Missing import of is_system_toolchain?

Reply:
No, we have both a method in the `Toolchain` class (which doesn't need an argument since a `Toolchain` instance knows its toolchain name), and a stand-alone function (which requires a toolchain name as argument, and which is used by `Toolchain.is_system_toolchain`)